.github/workflows/main.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  #
+  # Ensure the debian package can be built and installed
+  # without any errors. Note that we require python3 to
+  # be installed.
+  #
+  deb-build-and-test-run:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      #
+      # Needed for all the debian file related utilities.
+      #
+      - run: sudo apt install debhelper devscripts equivs
+      #
+      # Build depepndencies non-interactively.
+      #
+      - run: sudo mk-build-deps --install --tool='apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes' debian/control
+      #
+      # Create changelog.
+      #
+      # Note: The latest version of libkdumpfile has been 0.3.0
+      #       and has stayed such for many years, even though
+      #       the library has had multiple breaking changes. The
+      #       actual version number can be acquired through the
+      #       m4 macros defined in configure.ac (pkg_version_*).
+      #       Since there is not a clean/straightforward way to
+      #       get that value from the file we hardcode it here
+      #       for the changelog that we generate.
+      #
+      - run: dch --create --package "libkdumpfile" -v "0.3.0" "Auto-generated entry."
+      #
+      # Perform build and generate package.
+      #
+      - run: dpkg-buildpackage -b -us -uc
+      #
+      # Install the package.
+      #
+      - run: sudo dpkg -i ../libkdumpfile_*deb
```

Comment:
`dpkg -i` doesn't install any of the package's dependencies. I'd suggest to use `apt-get install` instead.